src/components/primary-toolbar/index.js

```diff
@@ -46,59 +30,9 @@ export const PrimaryToolbar = ({
         icon={MenuIcon}
         labelText={`${visible.sidebar ? 'Hide' : 'Show'} menu`}
       />
-      {isExperimentView ? null : (
-        <>
-          <IconButton
-            ariaLive="polite"
-            className={'pipeline-menu-button--labels'}
-            onClick={() => onToggleTextLabels(!textLabels)}
-            icon={LabelIcon}
-            labelText={`${textLabels ? 'Hide' : 'Show'} text labels`}
-            visible={visible.labelBtn}
-          />
-          <IconButton
-            ariaLabel={`Turn data layers ${visibleLayers ? 'off' : 'on'}`}
-            className={'pipeline-menu-button--layers'}
-            onClick={() => onToggleLayers(!visibleLayers)}
-            icon={LayersIcon}
-            labelText={`${visibleLayers ? 'Hide' : 'Show'} layers`}
-            disabled={disableLayerBtn}
-            visible={visible.layerBtn}
-          />
-          <IconButton
-            ariaLabel="Export graph as SVG or PNG"
-            className={'pipeline-menu-button--export'}
-            onClick={() => onToggleExportModal(true)}
-            icon={ExportIcon}
-            labelText="Export visualisation"
-            visible={visible.exportBtn}
-          />
-        </>
-      )}
+      {children}
     </ul>
   </>
 );
 
-export const mapStateToProps = (state) => ({
-  disableLayerBtn: !state.layer.ids.length,
-  textLabels: state.textLabels,
-  visible: state.visible,
-  visibleLayers: Boolean(getVisibleLayerIDs(state).length),
-});
-
-export const mapDispatchToProps = (dispatch) => ({
-  onToggleExportModal: (value) => {
-    dispatch(toggleExportModal(value));
-  },
-  onToggleLayers: (value) => {
-    dispatch(toggleLayers(Boolean(value)));
-  },
-  onToggleSidebar: (visible) => {
-    dispatch(toggleSidebar(visible));
-  },
-  onToggleTextLabels: (value) => {
-    dispatch(toggleTextLabels(Boolean(value)));
-  },
-});
-
-export default connect(mapStateToProps, mapDispatchToProps)(PrimaryToolbar);
+export default PrimaryToolbar;
```

Comment:
Should we rename this file and component to `BasePrimaryToolbar` or `PrimaryToolbarBase`? 

I think that could add a little bit of clarity and qualification, especially since the other`PrimaryToolbar` components (flowchart and experiment) have a descriptor.

Reply:
In looking at our codebase, this isn't a naming convention we follow - our reusable components adopts the typical naming as is. 

In this case, the descriptor is only added to specific setup (`experiment-tracking-primary-toolbar` and `flowchart-primary-toolbar`), which this setup is self-explanatory enough to differentiate the intent of the Primary toolbar? 

We should also revisit our current component file setup so we can move reusable components into a separate folder, which by then this will be even more explantory.